examples/openai-opentelemetry-example/setup_otel.go

```diff
@@ -0,0 +1,133 @@
+package main
```

Comment:
I would move this file to a package or inline the function into  the example go. This allows people to run the example both via `go run .` which works now, and `go run openai_chat_example.go` which doesn't work now.

Notably, this helps in IDEs where you select a file and click run. All of the other examples are single-file, which may be just by accident, or to prevent problems running.

For example, this works now:
<img width="916" alt="Screenshot 2024-08-21 at 2 06 55 PM" src="https://github.com/user-attachments/assets/9863c607-7d1a-4369-95d7-1cbe25fb1aaa">

But more of the time, people select the file they want to run, and doing so will fail as it doesn't consider setup_otel.go

<img width="825" alt="Screenshot 2024-08-21 at 2 07 50 PM" src="https://github.com/user-attachments/assets/a6c7ab6c-c6f9-4b56-9c12-b20cca6eed6f">